src/components/RadioButton/RadioButton.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import Text from '../Text';
+import styles from './RadioButton.scss';
+import { CHECKBOX_VIEWBOX } from '../../constants';
+
+const { block, elem } = bem({
+    name: 'RadioButton',
+    classnames: styles,
+    propsToMods: ['disabled']
+});
+
+const RadioButton = props => {
```

Comment:
@eszthoff Ideally we should also start forwarding refs, at least to `input`'s and `button`'s

https://reactjs.org/docs/forwarding-refs.html

Reply:
Let's discuss it together for all components, and see if this is a pattern we should make standard for all components we make. Then we can add it everywhere.
Thinking on it, we should have a developer's guide where we document all these standards, including how to use ...rest, and other things we agreed upon.

Reply:
Ok, let's keep this for later so that we can apply it to all components for which it makes sense.